src/web/assets/cp/src/js/BaseElementSelectInput.js

```diff
@@ -853,7 +879,7 @@ Craft.BaseElementSelectInput = Garnish.Base.extend(
       }, 200);
     },
 
-    selectElements: async function (elements) {
```

Comment:
@brianjhanson why did you remove `async`? Everywhere this method is called, a promise is expected.

Reply:
@brandonkelly my bad, at one point I saw this in the heap and was wondering if something was waiting on the `async`. I just missed putting it back before pushing it up.